src/components/KymaModules/KymaModulesCreate.js

```diff
@@ -361,8 +459,25 @@ export default function KymaModulesCreate({ resource, ...props }) {
         createUrl={props.resourceUrl}
         disableDefaultFields
         skipCreateFn={() => {
-          if (isEdited && !showMessageBox.hide) {
-            setShowMessageBox({ ...showMessageBox, isOpen: true, hide: true });
+          if (
+            (isEdited && !showMessageBox.hide) ||
+            (isManagedChanged && !showManagedBox.hide)
+          ) {
+            if (isManagedChanged && !showManagedBox.hide) {
+              setShowManagedBox({
+                ...showManagedBox,
+                isOpen: true,
+                hide: true,
+                onSave: true,
+              });
+            }
+            if (isEdited && !showMessageBox.hide) {
+              setShowMessageBox({
+                ...showMessageBox,
+                isOpen: true,
+                hide: true,
+              });
+            }
             return true;
           }
```

Comment:
I'd extract this fellow into a callback. Also can we somehow move `!showMessageButton.hide` above, as it's used 4 times here and there?